Include/object.h

```diff
@@ -142,8 +142,11 @@ static inline Py_region_ptr_with_tags_t Py_region_ptr_with_tags(Py_region_ptr_t
     return (Py_region_ptr_with_tags_t) { region };
 }
 
+int _Py_is_bridge_object(PyObject *op);
```

Comment:
A macro could make the usage slightly easier and more consistent with other functions
```suggestion
int _Py_is_bridge_object(PyObject *op);
#define Py_is_bridge_object(op) (_Py_is_bridge_object(_PyObject_CAST(op)))
```